### PR TITLE
Switch the location of POI data from metadata to DE

### PR DESCRIPTION
**Pull request by vilianvenkov**
Since the metadata is easily editable we need a better place to locate the POI.
We are moving the data to Define Entities, where we can have some access control.

Signed-off-by: Vilian Venkov <vvenkov@vmware.com>
Co-authored-by: Vilian Venkov <vvenkov@vmware.com>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
   ],
   "devDependencies": {
     "lerna": "^3.22.1",
-    "@types/yeoman-generator": "^3.0.2"
+    "@types/yeoman-generator": "^5.2.7"
```
Comment (stoychevl) [inline package.json:11]: Why put this on in the root level package.json? Should it be in "cli/package.json"
Also, shouldn't ideally the @types package version match the version of the yeoman-generator?

File: packages/care-package/abstract-plugin/src/ProviderOrg.ts [hunk 2]
```diff
@@ -1,31 +1,47 @@
-import {CloudDirectorConfig, OrgVdcNetworkApi, VdcApi} from '@vcd/node-client';
```
Comment (stoychevl) [inline packages/care-package/abstract-plugin/src/ProviderOrg.ts:1]: Do we really want this common provider org code to live under abstract-plugin?

File: packages/care-package/abstract-plugin/src/ProviderOrg.ts [hunk 3]
```diff
@@ -1,31 +1,47 @@
...
-    };
-    vdc: {
         name: string;
-        id: string;
+        capabilities: string[];
+        vdcs: {
+            id: string;
+            name: string;
+            capabilities: string[];
+            networks: {
```
Comment (stoychevl) [inline packages/care-package/abstract-plugin/src/ProviderOrg.ts:23]: Can we give this a type? It is used a couple of times.

File: packages/care-package/abstract-plugin/src/ProviderOrg.ts [hunk 4]
```diff
@@ -36,110 +52,90 @@ export class ProviderOrg {
...
+            throw new Error('Provider Organization Interface is not configured.');
+        }
+        log('DefinedEntity of ProviderOrg Interface found.');
+
+        const definedEntity = definedEntities.body.values.pop();
+        const providerOrgInterface = definedEntity.entity as ProviderOrgInterface;
+        const providerOrgApiConfig = this.apiConfig.actAs(providerOrgInterface.organization.id);
+        providerOrgInterface.organization.name = await ProviderOrg.getOrg(this.apiConfig, providerOrgInterface.organization.id);
+        log(`Organization: ${providerOrgInterface.organization.name} found.`);
+        for (const vdc of providerOrgInterface.organization.vdcs) {
```
Comment (stoychevl) [inline packages/care-package/abstract-plugin/src/ProviderOrg.ts:123]: Can't we either fetch these in parallel or use query by all IDs?
Same for all for loops bellow ?

File: packages/care-package/abstract-plugin/src/ProviderOrg.ts (hunk 2 above)
Comment (vilianvenkov) [reply packages/care-package/abstract-plugin/src/ProviderOrg.ts:1]: Future plugins might use it as well, so a common place is expected.

File: package.json (hunk 1 above)
Comment (vilianvenkov) [reply package.json:11]: setting it to each sub-package.json to the following version
"@types/yeoman-generator": "^4.11.0",

**Pull request merged**